sdk-workflows/src/test/java/io/dapr/workflows/client/DaprWorkflowClientTest.java

```diff
@@ -0,0 +1,31 @@
+package io.dapr.workflows.client;
+
+import com.microsoft.durabletask.DurableTaskClient;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class DaprWorkflowClientTest {
+  private DaprWorkflowClient client;
+
+  @Before
+  public void setUp() throws Exception {
+    DurableTaskClient innerClient = mock(DurableTaskClient.class);
+    when(innerClient.scheduleNewOrchestrationInstance(any())).thenReturn("TestWorkflowInstanceId");
+    client = new DaprWorkflowClient(innerClient);
+  }
+
+  @Test
+  public void scheduleNewWorkflow() {
+    Assert.assertNotNull(client.scheduleNewWorkflow("TestWorkflow"));
```

Comment:
Consider only creating the mock innerClient in the setup and using this test to make a stronger assertion like:

```java
String expectedInstanceId = "TestWorkflowInstanceId";
when(innerClient.scheduleNewOrchestrationInstance(any())).thenReturn(expectedInstanceId);
client = new DaprWorkflowClient(innerClient);
Assert.assertEquals(expectedInstanceId, client.scheduleNewWorkflow("TestWorkflow"));
```

Reply:
Valid. Adapting!